mmcls/models/backbones/efficientnet.py

```diff
@@ -0,0 +1,421 @@
+            Defaults to -1, which means not freezing any parameters.
+        conv_cfg (dict): Config dict for convolution layer.
+            Defaults to None, which means using conv2d.
+        norm_cfg (dict): Config dict for normalization layer.
+            Defaults to dict(type='BN').
+        act_cfg (dict): Config dict for activation layer.
+            Defaults to dict(type='Swish').
+        norm_eval (bool): Whether to set norm layers to eval mode, namely,
+            freeze running stats (mean and var). Note: Effect on Batch Norm
+            and its variants only. Defaults to False.
+        with_cp (bool): Use checkpoint or not. Using checkpoint will save some
+            memory while slowing down the training speed. Defaults to False.
+    """
+
+    # Parameters to build layers.
+    # 'b' represents the architecture of normal EfficientNet family includes
+    # 'b0', 'b1', 'b2', 'b3', 'b4', 'b5', 'b6', 'b7', 'b8'.
+    # 'e' represents the architecture of EfficientNet-EdgeTPU including 'es',
+    # 'em', 'el'.
+    # 9 parameters are needed to construct a layer, From left to right:
+    # kernel_size, out_channel, se_ratio, use_swish, stride, expand_ratio,
+    # block_type.
+    layer_settings = {
+        'b': [[[3, 32, 0, 1, 2, 0, -1]],
+              [[3, 16, 4, 1, 1, 1, 0]],
+              [[3, 24, 4, 1, 2, 6, 0],
+               [3, 24, 4, 1, 1, 6, 0]],
+              [[5, 40, 4, 1, 2, 6, 0],
+               [5, 40, 4, 1, 1, 6, 0]],
+              [[3, 80, 4, 1, 2, 6, 0],
+               [3, 80, 4, 1, 1, 6, 0],
+               [3, 80, 4, 1, 1, 6, 0],
+               [5, 112, 4, 1, 1, 6, 0],
+               [5, 112, 4, 1, 1, 6, 0],
+               [5, 112, 4, 1, 1, 6, 0]],
+              [[5, 192, 4, 1, 2, 6, 0],
+               [5, 192, 4, 1, 1, 6, 0],
+               [5, 192, 4, 1, 1, 6, 0],
+               [5, 192, 4, 1, 1, 6, 0],
+               [3, 320, 4, 1, 1, 6, 0]],
+              [[1, 1280, 0, 1, 1, 0, -1]]
+              ],
+        'e': [[[3, 32, 0, 0, 2, 0, -1]],
+              [[3, 24, 0, 0, 1, 3, 1]],
+              [[3, 32, 0, 0, 2, 8, 1],
+               [3, 32, 0, 0, 1, 8, 1]],
+              [[3, 48, 0, 0, 2, 8, 1],
+               [3, 48, 0, 0, 1, 8, 1],
+               [3, 48, 0, 0, 1, 8, 1],
+               [3, 48, 0, 0, 1, 8, 1]],
+              [[5, 96, 0, 0, 2, 8, 0],
+               [5, 96, 0, 0, 1, 8, 0],
+               [5, 96, 0, 0, 1, 8, 0],
+               [5, 96, 0, 0, 1, 8, 0],
+               [5, 96, 0, 0, 1, 8, 0],
+               [5, 144, 0, 0, 1, 8, 0],
+               [5, 144, 0, 0, 1, 8, 0],
+               [5, 144, 0, 0, 1, 8, 0],
+               [5, 144, 0, 0, 1, 8, 0]],
+              [[5, 192, 0, 0, 2, 8, 0],
+               [5, 192, 0, 0, 1, 8, 0]],
+              [[1, 1280, 0, 0, 1, 0, -1]]
+              ]
+    }  # yapf: disable
+
+    # Parameters to build different kinds of architecture.
+    # From left to right: scaling factor for width, scaling factor for depth,
```

Comment:
The statement of width and depth is not very intuitive, it is recommended to supplement.

Reply:
The width and depth are explained in the image of README